stylus/settings/breakpoints.styl

```diff
@@ -41,7 +44,7 @@ extra-large = 1439
     Styleguide Settings.breakpoints.tiny
 */
 tiny-screen(direction='max')
-    @media ({direction}-width: (tiny/basefont)rem)
+    @media ({direction}-width: (size-helper(direction, tiny)/basefont)rem)
```

Comment:
no need for `/basefont` with `rem`s right ?

Reply:
Je laisse répondre @GoOz la dessus ;). J'en ai aucune idée.

Reply:
non pardon je me suis trompé

Reply:
Bah non, transformer les `px` en `rem` sur la base du font-size c'est bien. :)